marketplace/dev/setup_config.sh

```diff
@@ -1,13 +1,29 @@
 #!/bin/bash
```

Comment:
How did you get github to recognize this move?

Reply:
I didn't do anything special. Probably it was moved with little diff in a commit, hence it was recognized.

Reply:
Nice. :)